frontend/src/i18n/en-US.json

```diff
@@ -37,9 +37,11 @@
   "admin.licenseInfo.expiresAt.description.valid": "Your license is valid.",
   "admin.licenseInfo.expiresAt.description.expired": "Your license has expired.",
   "admin.licenseInfo.manageSubscription": "Manage Subscription",
+  "admin.licenseInfo.type.title": "Type",
+  "admin.licenseInfo.upgradeLicense": "Get License",
   "admin.licenseInfo.communityLicense.description": "Thank you for using Cryptomator Hub! You have been granted the Community License. If you need more seats, upgrade your license.",
```

Comment:
Rename `admin.licenseInfo.communityLicense.description` to `admin.licenseInfo.selfHostedNoLicense.description`?

Reply:
Fixed in https://github.com/cryptomator/hub/pull/174/commits/9a7e1d46f452775cccec5351b8291a600f66f10e